src/ChatCore.ts

```diff
@@ -23,12 +28,12 @@ export class ChatCore {
   private endpoints: Endpoints;
   private promptPackage: ChatPrompt;
 
```

Comment:
can we update the doc for this class -- it's not the entry point of the library anymore, the provider layer is. and include a `@remarks` tag that this is consider internal and to encourage user to use the provider layer